playbooks/roles/mysql/tasks/main.yml

```diff
@@ -21,4 +21,12 @@
   when: remove_experimental_mysql
 
 - include: mysql.yml
-  when: (mysql_56_installed.rc == 1) or (remove_experimental_mysql)
\ No newline at end of file
+  when: (mysql_56_installed.rc == 1) or (remove_experimental_mysql)
+
+- name: Create remote root user
```

Comment:
@melvinsoft Do you intend for Insights to use its own local MySQL database for Basic tier? If we're using the remote database for Pro and Enterprise, we should probably do the same on Basic to make tier upgrades easier.